cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyUtilsTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.detector;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+import org.junit.Test;
+
+import static com.linkedin.kafka.cruisecontrol.detector.AnomalyUtils.buildTopicRegex;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+/**
+ * Unit test class for anomaly utilization methods.
+ */
+public class AnomalyUtilsTest {
+    private static String TOPIC1 = "abc";
+    private static String TOPIC2 = "def";
+    private static String TOPIC3 = "ghi";
+
+    @Test
+    public void testBuildTopicRegex() {
```

Comment:
The corresponding unit test seems to pass without parentheses, as well. `Capturing group` seems redundant.